client/app/states/services/details/details.html

```diff
@@ -79,6 +79,7 @@
                   <!--<a href="#" title="Suspend this service" ng-click="vm.handlePowerOperation('suspend', vm.service)">{{'Suspend'|translate}}</a>-->
                 <!--</li>-->
                 <li><a href="#" title="Edit this service" ng-click="vm.editServiceModal()" ng-show="vm.showEditService">{{'Edit'|translate}}</a></li>
+                <li><a href="#" title="Edit tags" ng-click="vm.tagEditorModal()" ng-show="vm.showTagEditor">{{'Edit Tags'|translate}}</a></li>
```

Comment:
Good call out, I think any user visible string should probably be translated. Probably worthy of a larger discussion, but I'm thinking we should be more consistent with our use of the `title` attribute in general. For consistency purposes we should probably not use `title` directly and instead use `uib-tooltip`s.